CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -24,6 +24,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### Other
 - [Remove nsswitch.conf from docker image](https://github.com/etcd-io/etcd/pull/15161)
+- [Fixes: All docker images of Architecture show amd64](https://github.com/etcd-io/etcd/pull/15612)
```

Comment:
Thanks @zhangguanzhang, suggest a wording tweak to align more with other entries:

```suggestion
- Fix [etcd docker images all tagged with amd64 architecture](https://github.com/etcd-io/etcd/pull/15612)
```

Reply:
Done, thanks